core/src/navigation/TopNav.html

```diff
@@ -279,6 +279,7 @@
             contextSwitcherToggle="{contextSwitcherToggle}"
             bind:selectedLabel="{selectedLabel}"
             bind:defaultLabel="{defaultLabelContextSwitcher}"
+            bind:nodePath="{nodePath}"
```

Comment:
```suggestion
```
Seems like no need to bind here. **TopNav** doesn't seem to use `nodePath` anywhere.